package/gluon-respondd/files/etc/hotplug.d/iface/10-gluon-respondd

```diff
@@ -6,29 +6,17 @@
 DEVLIST=/var/run/gluon-respondd.devs
 
 ifname_to_dev () {
-	json_load "$(ubus call network.interface.$1 status)"
-	json_get_var dev device
+  json_load "$(ubus call network.interface.$1 status)"
+  json_get_var dev device
```

Comment:
I think it would be nice to rewrite this to use *ifstatus* and *jsonfilter*, as the json_* functions are very slow.